content/en/security_monitoring/detection_rules/security_monitoring.md

```diff
@@ -44,6 +44,10 @@ Add additional queries with the Add Query button.
 
 **Note**: The query applies to all Datadog events and ingested logs which do not require indexing.
 
+#### Advanced options
+
+Click the **Advanced** option to add queries that will **Only trigger a signal when** a value is met, or **Never trigger a signal** when a value is met. For example, if a user is triggering a signal, but their actions are benign and no longer want signals triggered from this user, create a logs query that excludes `@user.username: john.doe` under `Never Trigger A Signal`.
```

Comment:
```suggestion
Click the **Advanced** option to add queries that will **Only trigger a signal** when a value is met, or **Never trigger a signal** when a value is met. For example, if a user is triggering a signal, but their actions are benign and you no longer want signals triggered from this user, create a logs query that excludes `@user.username: john.doe` under **Never Trigger A Signal**.
```
I think? Not sure where the word "when" belongs in each of these, inside or outside the bold tags. And not sure about monospace tagging on a UI element.

Reply:
Good catch on the UI element, definitely should've just been bold. Confirmed, it's **Never trigger a signal when:** in the UI, so I'll update.